account-compression/programs/wrapper/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "wrapper"
```

Comment:
is this meant to be published? If so, maybe spl-data-wrapper or something?

Reply:
This is not meant to be published. Would prefer to limit the "official" visibility of this package / program because, like you mentioned, this is currently a hack.